flavorite_OAS_v1.yaml

```
@@ -0,0 +1,639 @@
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: Review not found
+      security:
+        - api_key: []
+        - flavorite_auth:
+            - write:reviews
+            - read:reviews
+    delete:
+      tags:
+        - review
+      summary: Deletes a review
+      description: delete a review
+      operationId: deleteReview
+      parameters:
+        - name: api_key
+          in: header
+          description: ''
+          required: false
+          schema:
+            type: string
+        - name: reviewId
+          in: path
+          description: Review id to delete
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '400':
+          description: Invalid review value
+      security:
+        - flavorite_auth:
+            - write:reviews
+            - read:reviews
+            
+  /review/{userId}:
+    get:
+      tags:
+        - review
+      summary: Find reviews by user
+      description: Returns all reviews by a single user
+      operationId: getReviewByUser
+      parameters:
+        - name: userId
+          in: path
+          description: ID of user to return user's reviews
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Review'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/Review'
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: Review not found
+      security:
+        - api_key: []
+        - flavorite_auth:
+            - write:reviews
+            - read:reviews
+  /review/{restaurantId}:
+    get:
+      tags:
+        - review
+      summary: Find reviews by restaurant
+      description: Returns all reviews for a single restaurant
+      operationId: getReviewByRestaurant
+      parameters:
+        - name: restaurantId
+          in: path
+          description: ID of restaurant to return all reviews for a single restaurant
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Review'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/Review'
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: Review not found
+      security:
+        - api_key: []
+        - flavorite_auth:
+            - write:reviews
+            - read:reviews
+
+  /review/{reviewId}/uploadImage:
```

No camelCase in url.
```suggestion
  /review/{reviewId}/upload-image:
```